sympy/assumptions/tests/test_failing_relational_queries.py

```diff
@@ -0,0 +1,37 @@
+from sympy.testing.pytest import XFAIL
+from sympy.assumptions.ask import ask, Q
+from sympy.logic.boolalg import Equivalent, Implies, And, Or
+from sympy.abc import x, y, z
+
+
+
+@XFAIL
+def test_inequalities():
+    # test basic defintions
+    assert ask(Equivalent(Q.gt(x, 0), Q.positive(x))) is True
+    assert ask(Equivalent(Q.eq(x, 0), Q.zero(x))) is True
+    assert ask(Equivalent(Q.lt(x, 0), Q.negative(x))) is True
+    assert ask(Equivalent(Q.ge(x, 0), Or(Q.positive(x), Q.zero(x)))) is True
+    assert ask(Equivalent(Q.le(x, 0), Or(Q.negative(x), Q.zero(x)))) is True
+    assert ask(Implies(And(Q.gt(x, y), Q.positive(y)), Q.positive(x))) is True
+    assert ask(Implies(And(Q.lt(x, y), Q.negative(y)), Q.negative(x))) is True
+
+    # test more complex problems
+    assert ask(x > z, x > y & y > z)
```

Comment:
& has a higher precedence than > so you have to parenthesize things like this. 

```suggestion
    assert ask(x > z, (x > y) & (y > z))
```